scripts/linting.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+set -euo pipefail
+
+trap "exit -1" ERR
+
+# Track whether linting failed; we don't want to bail on lint failures
+failed=0
+
+## Main ----------------------------------------------------------------------
+echo "Running Basic Ansible Lint Check"
+
+
+# Install the development requirements.
+if [ -f "os-ansible-deployment/dev-requirements.txt" ]; then
+  pip2 install -r os-ansible-deployment/dev-requirements.txt || pip install -r os-ansible-deployment/dev-requirements.txt
+else
+  pip2 install ansible-lint || pip install ansible-lint
+fi
+
+# Run hacking/flake8 check for all python files
+# Ignores the following rules due to how ansible modules work in general
+#     F403 'from ansible.module_utils.basic import *' used; unable to detect undefined names
+#     H303  No wildcard (*) import.
+flake8 --ignore=F403,H303 $(grep --exclude-dir=os-ansible-deployment -rln -e '^#!/usr/bin/env python' -e '^#!/bin/python' -e '^#!/usr/bin/python' * ) || failed=1
```

Comment:
I'd really like us to exclude `scripts/f5-config.py` here. It's not really ours, we just vendor it, and it's full of lint problems that we'll just have to re-fix everytime we import it if we don't exclude it.